Import failing on macOS=14.7.4 with `/usr/lib/libabsl_log_internal_message.2407.0.0.dylib' (no such file, not in dyld cache)`

### Solution to issue cannot be found in the documentation.

- [x] I checked the documentation.

### Issue

Hi folks, got a weird issue that's been creeping up only since a few days now:
```
>   from pys2index import S2PointIndex
E   ImportError: dlopen(/Users/runner/miniconda3/envs/esmvaltool/lib/python3.12/site-packages/pys2index.cpython-312-darwin.so, 0x0002): Library not loaded: @rpath/libabsl_log_internal_message.2407.0.0.dylib
E     Referenced from: <CF872AB1-3C47-3526-87E1-A644AE4AF3BD> /Users/runner/miniconda3/envs/esmvaltool/lib/python3.12/site-packages/pys2index.cpython-312-darwin.so
E     Reason: tried: '/Users/runner/miniconda3/envs/esmvaltool/lib/python3.12/site-packages/../../libabsl_log_internal_message.2407.0.0.dylib' (no such file), '/Users/runner/miniconda3/envs/esmvaltool/lib/python3.12/site-packages/../../libabsl_log_internal_message.2407.0.0.dylib' (no such file), '/Users/runner/miniconda3/envs/esmvaltool/bin/../lib/libabsl_log_internal_message.2407.0.0.dylib' (no such file), '/Users/runner/miniconda3/envs/esmvaltool/bin/../lib/libabsl_log_internal_message.2407.0.0.dylib' (no such file), '/usr/local/lib/libabsl_log_internal_message.2407.0.0.dylib' (no such file), '/usr/lib/libabsl_log_internal_message.2407.0.0.dylib' (no such file, not in dyld cache)
```

see reliably failing test (ie no flakiness) at our [GHA](https://github.com/ESMValGroup/ESMValTool/actions/runs/14049210059/job/39336299826) - unfortunately we can't replicate it locally since none of us has a Mac anymore (we're all Linux). I'll attach the relevant versions below, also for a full list of deps in the env, pls have a look at the [GHA](https://github.com/ESMValGroup/ESMValTool/actions/runs/14049210059/job/39336299826) and click the "Inspect environment" step (same deps as below tho). Cheers muchly for looking at it 🍻  

### Installed packages

```shell
_py-xgboost-mutex         2.0                       cpu_0    conda-forge
accessible-pygments       0.0.5              pyhd8ed1ab_1    conda-forge
adwaita-icon-theme        48.0                     unix_0    conda-forge
affine                    2.4.0              pyhd8ed1ab_1    conda-forge
aiohappyeyeballs          2.6.1              pyhd8ed1ab_0    conda-forge
aiohttp                   3.11.14         py312h3520af0_0    conda-forge
aiosignal                 1.3.2              pyhd8ed1ab_0    conda-forge
alabaster                 1.0.0              pyhd8ed1ab_1    conda-forge
aom                       3.9.1                hf036a51_0    conda-forge
astroid                   3.3.9           py312hb401068_0    conda-forge
asttokens                 3.0.0              pyhd8ed1ab_1    conda-forge
atk-1.0                   2.38.0               h4bec284_2    conda-forge
attrs                     25.3.0             pyh71513ae_0    conda-forge
autodocsumm               0.2.14             pyhd8ed1ab_0    conda-forge
aws-c-auth                0.8.6                h321fff7_4    conda-forge
aws-c-cal                 0.8.7                hfaf822f_0    conda-forge
aws-c-common              0.12.0               h6e16a3a_0    conda-forge
aws-c-compression         0.3.1                hb1ee187_2    conda-forge
aws-c-event-stream        0.5.4                hf9b3e9c_2    conda-forge
aws-c-http                0.9.4                h29be59e_4    conda-forge
aws-c-io                  0.17.0               h786d7a7_6    conda-forge
aws-c-mqtt                0.12.2               h6a909e1_2    conda-forge
aws-c-s3                  0.7.13               h2313cb2_2    conda-forge
aws-c-sdkutils            0.2.3                hb1ee187_2    conda-forge
aws-checksums             0.2.3                hb1ee187_2    conda-forge
aws-crt-cpp               0.31.0               hc7e8f17_4    conda-forge
aws-sdk-cpp               1.11.510             ha0394b9_3    conda-forge
azure-core-cpp            1.14.0               h9a36307_0    conda-forge
azure-identity-cpp        1.10.0               ha4e2ba9_0    conda-forge
azure-storage-blobs-cpp   12.13.0              h3d2f5f1_1    conda-forge
azure-storage-common-cpp  12.8.0               h1ccc5ac_1    conda-forge
azure-storage-files-datalake-cpp 12.12.0              h86941f0_1    conda-forge
babel                     2.17.0             pyhd8ed1ab_0    conda-forge
backports.zoneinfo        0.2.1           py312hb401068_9    conda-forge
beautifulsoup4            4.13.3             pyha770c72_0    conda-forge
bleach                    6.2.0              pyh29332c3_4    conda-forge
bleach-with-css           6.2.0                h82add2a_4    conda-forge
blosc                     1.21.6               hd145fbb_1    conda-forge
bokeh                     3.7.0              pyhd8ed1ab_0    conda-forge
brotli                    1.1.0                h00291cd_2    conda-forge
brotli-bin                1.1.0                h00291cd_2    conda-forge
brotli-python             1.1.0           py312h5861a67_2    conda-forge
brunsli                   0.1                  h046ec9c_0    conda-forge
bzip2                     1.0.8                hfdf4475_7    conda-forge
c-ares                    1.34.4               hf13058a_0    conda-forge
c-blosc2                  2.15.2               h62acda9_1    conda-forge
ca-certificates           2025.1.31            h8857fd0_0    conda-forge
cairo                     1.18.4               h950ec3b_0    conda-forge
cartopy                   0.24.0          py312h98e817e_0    conda-forge
cattrs                    24.1.2             pyhd8ed1ab_1    conda-forge
cdo                       2.4.4                h0a4ec60_2    conda-forge
cdsapi                    0.7.5              pyhd8ed1ab_1    conda-forge
certifi                   2025.1.31          pyhd8ed1ab_0    conda-forge
cf-units                  3.3.0           py312h025c719_0    conda-forge
cf_xarray                 0.10.4             pyhd8ed1ab_0    conda-forge
cffi                      1.17.1          py312hf857d28_0    conda-forge
cfgrib                    0.9.15.0           pyhd8ed1ab_0    conda-forge
cfgv                      3.3.1              pyhd8ed1ab_1    conda-forge
cftime                    1.6.4           py312h3a11e2b_1    conda-forge
charls                    2.4.2                he965462_0    conda-forge
charset-normalizer        3.4.1              pyhd8ed1ab_0    conda-forge
chart-studio              1.1.0              pyhd8ed1ab_1    conda-forge
click                     8.1.8              pyh707e725_0    conda-forge
click-plugins             1.1.1              pyhd8ed1ab_1    conda-forge
cligj                     0.7.2              pyhd8ed1ab_2    conda-forge
cloudpickle               3.1.1              pyhd8ed1ab_0    conda-forge
cmocean                   4.0.3              pyhd8ed1ab_1    conda-forge
codespell                 2.3.0              pyhd8ed1ab_1    conda-forge
colorama                  0.4.6              pyhd8ed1ab_1    conda-forge
colorspacious             1.1.2              pyhecae5ae_1    conda-forge
config                    0.5.1              pyhd8ed1ab_1    conda-forge
configargparse            1.7                pyhd8ed1ab_1    conda-forge
contourpy                 1.3.1           py312hc47a885_0    conda-forge
coverage                  7.7.1           py312h3520af0_0    conda-forge
crc32c                    2.7.1           py312h01d7ebd_1    conda-forge
cryptography              44.0.2          py312h0995e51_0    conda-forge
cycler                    0.12.1             pyhd8ed1ab_1    conda-forge
cython                    3.0.12          py312hdfbeeba_0    conda-forge
cytoolz                   1.0.1           py312h01d7ebd_0    conda-forge
dask                      2025.3.0           pyhd8ed1ab_0    conda-forge
dask-core                 2025.3.0           pyhd8ed1ab_0    conda-forge
dask-jobqueue             0.9.0              pyhd8ed1ab_0    conda-forge
datapi                    0.3.0              pyhd8ed1ab_0    conda-forge
dav1d                     1.2.1                h0dc2134_0    conda-forge
decorator                 5.2.1              pyhd8ed1ab_0    conda-forge
defusedxml                0.7.1              pyhd8ed1ab_0    conda-forge
deprecated                1.2.18             pyhd8ed1ab_0    conda-forge
dill                      0.3.9              pyhd8ed1ab_1    conda-forge
distlib                   0.3.9              pyhd8ed1ab_1    conda-forge
distributed               2025.3.0           pyhd8ed1ab_0    conda-forge
docformatter              1.7.5              pyhd8ed1ab_1    conda-forge
docrep                    0.3.2              pyh44b312d_0    conda-forge
docutils                  0.21.2             pyhd8ed1ab_1    conda-forge
dodgy                     0.2.1                      py_0    conda-forge
donfig                    0.8.1.post1        pyhd8ed1ab_1    conda-forge
eccodes                   2.39.0               h18939f4_1    conda-forge
ecmwf-api-client          1.6.5              pyhd8ed1ab_0    conda-forge
eofs                      2.0.0              pyhff2d567_0    conda-forge
epoxy                     1.5.10               h5eb16cf_1    conda-forge
esgf-pyclient             0.3.1              pyhd8ed1ab_5    conda-forge
esmf                      8.7.0           nompi_h1438e41_1    conda-forge
esmpy                     8.7.0              pyhecae5ae_1    conda-forge
esmvalcore                2.12.0             pyhd8ed1ab_0    conda-forge
esmvaltool                2.13.0.dev10+g7883d411e          pypi_0    pypi
et_xmlfile                2.0.0              pyhd8ed1ab_1    conda-forge
exceptiongroup            1.2.2              pyhd8ed1ab_1    conda-forge
execnet                   2.1.1              pyhd8ed1ab_1    conda-forge
executing                 2.1.0              pyhd8ed1ab_1    conda-forge
expat                     2.6.4                h240833e_0    conda-forge
fftw                      3.3.10          nompi_h292e606_110    conda-forge
filelock                  3.18.0             pyhd8ed1ab_0    conda-forge
findlibs                  0.0.5              pyhd8ed1ab_0    conda-forge
fiona                     1.10.1          py312h4bcfd6b_3    conda-forge
fire                      0.7.0              pyhd8ed1ab_0    conda-forge
flake8                    7.1.2              pyhd8ed1ab_0    conda-forge
flake8-polyfill           1.0.2              pyhd8ed1ab_1    conda-forge
font-ttf-dejavu-sans-mono 2.37                 hab24e00_0    conda-forge
font-ttf-inconsolata      3.000                h77eed37_0    conda-forge
font-ttf-source-code-pro  2.038                h77eed37_0    conda-forge
font-ttf-ubuntu           0.83                 h77eed37_3    conda-forge
fontconfig                2.15.0               h37eeddb_1    conda-forge
fonts-conda-ecosystem     1                             0    conda-forge
fonts-conda-forge         1                             0    conda-forge
fonttools                 4.56.0          py312h3520af0_0    conda-forge
freetype                  2.13.3               h40dfd5c_0    conda-forge
freexl                    2.0.0                h3183152_2    conda-forge
fribidi                   1.0.10               hbcb3906_0    conda-forge
frozenlist                1.5.0           py312h3520af0_1    conda-forge
fsspec                    2025.3.0           pyhd8ed1ab_0    conda-forge
funcargparse              0.2.5              pyhd8ed1ab_0    conda-forge
gdal                      3.10.2          py312hd828770_4    conda-forge
gdk-pixbuf                2.42.12              ha587570_0    conda-forge
geographiclib             2.0                pyhd8ed1ab_1    conda-forge
geopy                     2.4.1              pyhd8ed1ab_2    conda-forge
geos                      3.13.1               h502464c_0    conda-forge
geotiff                   1.7.4                h0978cfe_0    conda-forge
gflags                    2.2.2             hac325c4_1005    conda-forge
giflib                    5.2.2                h10d778d_0    conda-forge
git                       2.49.0          pl5321h0e333bc_0    conda-forge
gitdb                     4.0.12             pyhd8ed1ab_0    conda-forge
gitpython                 3.1.44             pyhff2d567_0    conda-forge
glib-tools                2.82.2               hf8faeaf_1    conda-forge
glog                      0.7.1                h2790a97_0    conda-forge
graphite2                 1.3.13            h73e2aa4_1003    conda-forge
graphviz                  12.2.1               h44a0556_1    conda-forge
gtk3                      3.24.43              h82a860e_4    conda-forge
gts                       0.7.6                h53e17e3_4    conda-forge
h2                        4.2.0              pyhd8ed1ab_0    conda-forge
harfbuzz                  10.4.0               h86b413f_0    conda-forge
hdf4                      4.2.15               h8138101_7    conda-forge
hdf5                      1.14.4          nompi_h1607680_105    conda-forge
hicolor-icon-theme        0.17                 h694c41f_2    conda-forge
hpack                     4.1.0              pyhd8ed1ab_0    conda-forge
humanfriendly             10.0               pyh707e725_8    conda-forge
hyperframe                6.1.0              pyhd8ed1ab_0    conda-forge
icu                       75.1                 h120a0e1_0    conda-forge
identify                  2.6.9              pyhd8ed1ab_0    conda-forge
idna                      3.10               pyhd8ed1ab_1    conda-forge
imagecodecs               2024.12.30      py312h6a2ed5f_0    conda-forge
imagehash                 4.3.2              pyhd8ed1ab_0    conda-forge
imageio                   2.37.0             pyhfb79c49_0    conda-forge
imagesize                 1.4.1              pyhd8ed1ab_0    conda-forge
importlib-metadata        7.2.1              pyha770c72_0    conda-forge
importlib_metadata        7.2.1                hd8ed1ab_0    conda-forge
importlib_resources       6.5.2              pyhd8ed1ab_0    conda-forge
iniconfig                 2.0.0              pyhd8ed1ab_1    conda-forge
ipython                   8.34.0             pyh907856f_0    conda-forge
iris                      3.12.0             pyha770c72_0    conda-forge
iris-esmf-regrid          0.12.0             pyhd8ed1ab_0    conda-forge
iris-grib                 0.21.0             pyhd8ed1ab_0    conda-forge
isodate                   0.7.2              pyhd8ed1ab_1    conda-forge
isort                     6.0.1              pyhd8ed1ab_0    conda-forge
itsdangerous              2.2.0              pyhd8ed1ab_1    conda-forge
jasper                    4.2.4                hb10263b_0    conda-forge
jedi                      0.19.2             pyhd8ed1ab_1    conda-forge
jinja2                    3.1.6              pyhd8ed1ab_0    conda-forge
joblib                    1.4.2              pyhd8ed1ab_1    conda-forge
json-c                    0.18                 hc62ec3d_0    conda-forge
jsonschema                4.23.0             pyhd8ed1ab_1    conda-forge
jsonschema-specifications 2024.10.1          pyhd8ed1ab_1    conda-forge
jupyter_client            8.6.3              pyhd8ed1ab_1    conda-forge
jupyter_core              5.7.2              pyh31011fe_1    conda-forge
jupyterlab_pygments       0.3.0              pyhd8ed1ab_2    conda-forge
jxrlib                    1.1                  h10d778d_3    conda-forge
kiwisolver                1.4.8           py312h9275861_0    conda-forge
krb5                      1.21.3               h37d8d59_0    conda-forge
latexcodec                2.0.1              pyh9f0ad1d_0    conda-forge
lazy-loader               0.4                pyhd8ed1ab_2    conda-forge
lazy_loader               0.4                pyhd8ed1ab_2    conda-forge
lcms2                     2.17                 h72f5680_0    conda-forge
legacy-cgi                2.6.2              pyh41aed27_1    conda-forge
lerc                      4.0.0                hb486fe8_0    conda-forge
libabseil                 20250127.1      cxx17_h0e468a2_0    conda-forge
libaec                    1.1.3                h73e2aa4_0    conda-forge
libarchive                3.7.7                h1a33361_3    conda-forge
libarrow                  19.0.1           h13a0e53_5_cpu    conda-forge
libarrow-acero            19.0.1           hdc53af8_5_cpu    conda-forge
libarrow-dataset          19.0.1           hdc53af8_5_cpu    conda-forge
libarrow-substrait        19.0.1           ha37b807_5_cpu    conda-forge
libavif16                 1.2.1                hecd9f61_0    conda-forge
libblas                   3.9.0           31_h7f60823_openblas    conda-forge
libbrotlicommon           1.1.0                h00291cd_2    conda-forge
libbrotlidec              1.1.0                h00291cd_2    conda-forge
libbrotlienc              1.1.0                h00291cd_2    conda-forge
libcblas                  3.9.0           31_hff6cab4_openblas    conda-forge
libcrc32c                 1.1.2                he49afe7_0    conda-forge
libcurl                   8.12.1               h5dec5d8_0    conda-forge
libcxx                    20.1.1               hf95d169_0    conda-forge
libde265                  1.0.15               h7728843_0    conda-forge
libdeflate                1.23                 he65b83e_0    conda-forge
libedit                   3.1.20250104    pl5321ha958ccf_0    conda-forge
libev                     4.33                 h10d778d_2    conda-forge
libevent                  2.1.12               ha90c15b_1    conda-forge
libexpat                  2.6.4                h240833e_0    conda-forge
libffi                    3.4.6                h281671d_0    conda-forge
libgd                     2.3.3               h8555400_11    conda-forge
libgdal-core              3.10.2               h7c8127d_1    conda-forge
libgfortran               5.0.0           14_2_0_h51e75f0_1    conda-forge
libgfortran5              14.2.0               h51e75f0_1    conda-forge
libglib                   2.82.2               h5c976ab_1    conda-forge
libgoogle-cloud           2.36.0               h777fda5_1    conda-forge
libgoogle-cloud-storage   2.36.0               h3397294_1    conda-forge
libgrpc                   1.71.0               h53c9a1c_0    conda-forge
libheif                   1.19.7          gpl_h95ec88c_100    conda-forge
libhwy                    1.1.0                h7728843_0    conda-forge
libiconv                  1.18                 h4b5e92a_1    conda-forge
libintl                   0.23.1               h27064b9_0    conda-forge
libjpeg-turbo             3.0.0                h0dc2134_1    conda-forge
libjxl                    0.11.1               ha9171ad_0    conda-forge
libkml                    1.3.0             h9ee1731_1021    conda-forge
liblapack                 3.9.0           31_h236ab99_openblas    conda-forge
liblzma                   5.6.4                hd471939_0    conda-forge
libnetcdf                 4.9.2           nompi_hd7a758f_116    conda-forge
libnghttp2                1.64.0               hc7306c3_0    conda-forge
libopenblas               0.3.29          openmp_hbf64a52_0    conda-forge
libopentelemetry-cpp      1.19.0               h30c661f_0    conda-forge
libopentelemetry-cpp-headers 1.19.0               h694c41f_0    conda-forge
libparquet                19.0.1           h283e888_5_cpu    conda-forge
libpng                    1.6.47               h3c4a55f_0    conda-forge
libprotobuf               5.29.3               h1c7185b_0    conda-forge
libre2-11                 2024.07.02           h08ce7b7_3    conda-forge
librsvg                   2.58.4               h21a6cfa_2    conda-forge
librttopo                 1.1.0               hd2ea1e3_18    conda-forge
libsodium                 1.0.20               hfdf4475_0    conda-forge
libspatialite             5.1.0               ha73c641_13    conda-forge
libsqlite                 3.49.1               hdb6dae5_2    conda-forge
libssh2                   1.11.1               h3dc7d44_0    conda-forge
libthrift                 0.21.0               h75589b3_0    conda-forge
libtiff                   4.7.0                hb77a491_3    conda-forge
libudunits2               2.2.28               h516ac8c_3    conda-forge
libutf8proc               2.10.0               h777c5d8_0    conda-forge
libuuid                   2.38.1               hb7f2c08_0    conda-forge
libwebp-base              1.5.0                h6cf52b4_0    conda-forge
libxcb                    1.17.0               hf1f96e2_0    conda-forge
libxgboost                2.1.4            cpu_h250267a_1    conda-forge
libxml2                   2.13.6               hebb159f_0    conda-forge
libxslt                   1.1.39               h03b04e6_0    conda-forge
libzip                    1.11.2               h31df5bb_0    conda-forge
libzlib                   1.3.1                hd23fc13_2    conda-forge
libzopfli                 1.0.3                h046ec9c_0    conda-forge
lime                      0.2.0.1            pyhd8ed1ab_2    conda-forge
llvm-openmp               20.1.1               ha54dae1_1    conda-forge
llvmlite                  0.44.0          py312hc7f3abb_1    conda-forge
locket                    1.0.0              pyhd8ed1ab_0    conda-forge
logilab-common            1.7.3                      py_0    conda-forge
lxml                      5.3.1           py312h91b2f42_0    conda-forge
lz4                       4.3.3           py312h3d55e07_2    conda-forge
lz4-c                     1.10.0               h240833e_1    conda-forge
lzo                       2.10              h10d778d_1001    conda-forge
magics                    4.15.5               hb6eb63f_0    conda-forge
magics-python             1.5.8              pyhd8ed1ab_1    conda-forge
mapgenerator              1.0.7              pyhd8ed1ab_0    conda-forge
markdown-it-py            3.0.0              pyhd8ed1ab_1    conda-forge
markupsafe                3.0.2           py312h3520af0_1    conda-forge
matplotlib-base           3.10.1          py312h535dea3_0    conda-forge
matplotlib-inline         0.1.7              pyhd8ed1ab_1    conda-forge
mccabe                    0.7.0              pyhd8ed1ab_1    conda-forge
mdurl                     0.1.2              pyhd8ed1ab_1    conda-forge
minizip                   4.0.7                hfb7a1ec_3    conda-forge
mistune                   3.1.3              pyh29332c3_0    conda-forge
msgpack-python            1.1.0           py312hc5c4d5f_0    conda-forge
multidict                 6.2.0           py312h6f3313d_0    conda-forge
multiurl                  0.3.5              pyhd8ed1ab_0    conda-forge
munkres                   1.1.4              pyh9f0ad1d_0    conda-forge
myproxyclient             2.1.1              pyhd8ed1ab_1    conda-forge
mypy_extensions           1.0.0              pyha770c72_1    conda-forge
narwhals                  1.32.0             pyhd8ed1ab_0    conda-forge
natsort                   8.4.0              pyh29332c3_1    conda-forge
nbclient                  0.10.2             pyhd8ed1ab_0    conda-forge
nbconvert                 7.16.6               hb482800_0    conda-forge
nbconvert-core            7.16.6             pyh29332c3_0    conda-forge
nbconvert-pandoc          7.16.6               hed9df3c_0    conda-forge
nbformat                  5.10.4             pyhd8ed1ab_1    conda-forge
nbsphinx                  0.9.7              pyhd8ed1ab_0    conda-forge
nc-time-axis              1.4.1              pyhd8ed1ab_1    conda-forge
ncurses                   6.5                  h0622a9a_3    conda-forge
nested-lookup             0.2.25             pyhd8ed1ab_2    conda-forge
netcdf-fortran            4.6.1           nompi_h25c65ff_108    conda-forge
netcdf4                   1.7.2           nompi_py312haf04bd4_101    conda-forge
networkx                  3.4.2              pyh267e887_2    conda-forge
nlohmann_json             3.11.3               hf036a51_1    conda-forge
nodeenv                   1.9.1              pyhd8ed1ab_1    conda-forge
numba                     0.61.0          py312hfa89a5f_1    conda-forge
numcodecs                 0.15.1          py312hec45ffd_0    conda-forge
numpy                     2.1.3           py312hfc93d17_0    conda-forge
openjpeg                  2.5.3                h7fd6d84_0    conda-forge
openpyxl                  3.1.5           py312h732d5f6_1    conda-forge
openssl                   3.4.1                hc426f3f_0    conda-forge
orc                       2.1.1                h82caab2_1    conda-forge
packaging                 24.2               pyhd8ed1ab_2    conda-forge
pandas                    2.2.3           py312h98e817e_1    conda-forge
pandoc                    3.6.4                h694c41f_0    conda-forge
pandocfilters             1.5.0              pyhd8ed1ab_0    conda-forge
pango                     1.56.3               hf94f63b_0    conda-forge
parso                     0.8.4              pyhd8ed1ab_1    conda-forge
partd                     1.4.2              pyhd8ed1ab_0    conda-forge
pathspec                  0.12.1             pyhd8ed1ab_1    conda-forge
patsy                     1.0.1              pyhd8ed1ab_1    conda-forge
pcre2                     10.44                h7634a1b_2    conda-forge
pep8-naming               0.10.0             pyh9f0ad1d_0    conda-forge
perl                      5.32.1          7_h10d778d_perl5    conda-forge
pexpect                   4.9.0              pyhd8ed1ab_1    conda-forge
pickleshare               0.7.5           pyhd8ed1ab_1004    conda-forge
pillow                    11.1.0          py312hd9f36e3_0    conda-forge
pip                       25.0.1             pyh8b19718_0    conda-forge
pixman                    0.44.2               h1fd1274_0    conda-forge
pkgutil-resolve-name      1.3.10             pyhd8ed1ab_2    conda-forge
platformdirs              4.3.7              pyh29332c3_0    conda-forge
plotly                    6.0.1              pyhd8ed1ab_0    conda-forge
pluggy                    1.5.0              pyhd8ed1ab_1    conda-forge
pooch                     1.8.2              pyhd8ed1ab_1    conda-forge
pre-commit                4.2.0              pyha770c72_0    conda-forge
progressbar2              4.5.0              pyhd8ed1ab_1    conda-forge
proj                      9.5.1                h5273da6_0    conda-forge
prometheus-cpp            1.3.0                h7802330_0    conda-forge
prompt-toolkit            3.0.50             pyha770c72_0    conda-forge
propcache                 0.2.1           py312h3520af0_1    conda-forge
prospector                1.15.3             pyhd8ed1ab_0    conda-forge
prov                      2.0.0              pyhd3deb0d_0    conda-forge
psutil                    7.0.0           py312h01d7ebd_0    conda-forge
psy-maps                  1.5.0           py312hb401068_1    conda-forge
psy-reg                   1.5.0           py312hb401068_1    conda-forge
psy-simple                1.5.1           py312hb401068_0    conda-forge
psyplot                   1.5.1           py312hb401068_1    conda-forge
pthread-stubs             0.4               h00291cd_1002    conda-forge
ptyprocess                0.7.0              pyhd8ed1ab_1    conda-forge
pure_eval                 0.2.3              pyhd8ed1ab_1    conda-forge
py-cordex                 0.9.0              pyhd8ed1ab_1    conda-forge
py-xgboost                2.1.4           cpu_pyhf5c67e3_1    conda-forge
pyarrow                   19.0.1          py312hb401068_0    conda-forge
pyarrow-core              19.0.1          py312h5157fe3_0_cpu    conda-forge
pybtex                    0.24.0             pyhd8ed1ab_3    conda-forge
pycodestyle               2.12.1             pyhd8ed1ab_1    conda-forge
pycparser                 2.22               pyh29332c3_1    conda-forge
pydata-sphinx-theme       0.16.1             pyhd8ed1ab_0    conda-forge
pydocstyle                6.3.0              pyhd8ed1ab_1    conda-forge
pydot                     3.0.4           py312hb401068_0    conda-forge
pyflakes                  3.2.0              pyhd8ed1ab_1    conda-forge
pygments                  2.19.1             pyhd8ed1ab_0    conda-forge
pylint                    3.3.5              pyh29332c3_0    conda-forge
pylint-celery             0.3                        py_1    conda-forge
pylint-django             2.6.1              pyhd8ed1ab_1    conda-forge
pylint-flask              0.6                        py_0    conda-forge
pylint-plugin-utils       0.8.2              pyhd8ed1ab_1    conda-forge
pyopenssl                 25.0.0             pyhd8ed1ab_0    conda-forge
pyparsing                 3.2.2              pyhd8ed1ab_0    conda-forge
pyproj                    3.7.1           py312hc805472_0    conda-forge
pyproject_hooks           1.2.0              pyhd8ed1ab_1    conda-forge
pyroma                    4.2                pyhd8ed1ab_0    conda-forge
pys2index                 0.1.5           py312hae07849_2    conda-forge
pyshp                     2.3.1              pyhd8ed1ab_1    conda-forge
pysocks                   1.7.1              pyha55dd90_7    conda-forge
pytest                    8.3.5              pyhd8ed1ab_0    conda-forge
pytest-cov                6.0.0              pyhd8ed1ab_1    conda-forge
pytest-env                1.1.5              pyhd8ed1ab_1    conda-forge
pytest-html               4.1.1              pyhd8ed1ab_1    conda-forge
pytest-metadata           3.1.1              pyhd8ed1ab_1    conda-forge
pytest-mock               3.14.0             pyhd8ed1ab_1    conda-forge
pytest-xdist              3.6.1              pyhd8ed1ab_1    conda-forge
python                    3.12.9          h9ccd52b_1_cpython    conda-forge
python-build              1.2.2.post1        pyhff2d567_1    conda-forge
python-cdo                1.6.0              pyhff2d567_1    conda-forge
python-dateutil           2.9.0.post0        pyhff2d567_1    conda-forge
python-eccodes            2.37.0          py312h3a11e2b_0    conda-forge
python-fastjsonschema     2.21.1             pyhd8ed1ab_0    conda-forge
python-stratify           0.3.0           py312h3a11e2b_3    conda-forge
python-tzdata             2025.2             pyhd8ed1ab_0    conda-forge
python-utils              3.9.1              pyhff2d567_1    conda-forge
python-xxhash             3.5.0           py312h01d7ebd_2    conda-forge
python_abi                3.12                    5_cp312    conda-forge
pytz                      2024.1             pyhd8ed1ab_0    conda-forge
pywavelets                1.8.0           py312h025c719_0    conda-forge
pyyaml                    6.0.2           py312h3520af0_2    conda-forge
pyzmq                     26.3.0          py312h679dbab_0    conda-forge
qhull                     2020.2               h3c5361c_5    conda-forge
rasterio                  1.4.3           py312h7846a4c_0    conda-forge
rav1e                     0.6.6                h7205ca4_2    conda-forge
rdflib                    7.1.3              pyhd8ed1ab_0    conda-forge
re2                       2024.07.02           hf8a452e_3    conda-forge
readline                  8.2                  h7cca4af_2    conda-forge
referencing               0.36.2             pyh29332c3_0    conda-forge
requests                  2.32.3             pyhd8ed1ab_1    conda-forge
requests-cache            1.2.1              pyhd8ed1ab_1    conda-forge
requirements-detector     1.3.2              pyhd8ed1ab_1    conda-forge
retrying                  1.3.4              pyhd8ed1ab_0    conda-forge
rich                      13.9.4             pyhd8ed1ab_1    conda-forge
roman-numerals-py         3.1.0              pyhd8ed1ab_0    conda-forge
rpds-py                   0.23.1          py312hb59e30e_0    conda-forge
ruamel.yaml               0.18.10         py312h01d7ebd_0    conda-forge
ruamel.yaml.clib          0.2.8           py312h3d0f464_1    conda-forge
s2geometry                0.11.1               h1902187_4    conda-forge
scikit-image              0.25.2          py312hec45ffd_0    conda-forge
scikit-learn              1.6.1           py312he1a5313_0    conda-forge
scipy                     1.15.2          py312hd04560d_0    conda-forge
seaborn                   0.13.2               hd8ed1ab_3    conda-forge
seaborn-base              0.13.2             pyhd8ed1ab_3    conda-forge
seawater                  3.3.5              pyhd8ed1ab_1    conda-forge
semver                    3.0.4              pyhd8ed1ab_0    conda-forge
setoptconf-tmp            0.3.1              pyhd8ed1ab_0    conda-forge
setuptools                75.8.2             pyhff2d567_0    conda-forge
shapely                   2.0.7           py312hbf10b29_1    conda-forge
simplejson                3.20.1          py312h01d7ebd_0    conda-forge
six                       1.17.0             pyhd8ed1ab_0    conda-forge
smmap                     5.0.2              pyhd8ed1ab_0    conda-forge
snappy                    1.2.1                haf3c120_1    conda-forge
snowballstemmer           2.2.0              pyhd8ed1ab_0    conda-forge
snuggs                    1.4.7              pyhd8ed1ab_2    conda-forge
sortedcontainers          2.4.0              pyhd8ed1ab_1    conda-forge
soupsieve                 2.5                pyhd8ed1ab_1    conda-forge
sparse                    0.15.5             pyh72ffeb9_0    conda-forge
sphinx                    8.1.3                    pypi_0    pypi
sphinxcontrib-applehelp   2.0.0              pyhd8ed1ab_1    conda-forge
sphinxcontrib-devhelp     2.0.0              pyhd8ed1ab_1    conda-forge
sphinxcontrib-htmlhelp    2.1.0              pyhd8ed1ab_1    conda-forge
sphinxcontrib-jsmath      1.0.1              pyhd8ed1ab_1    conda-forge
sphinxcontrib-qthelp      2.0.0              pyhd8ed1ab_1    conda-forge
sphinxcontrib-serializinghtml 1.1.10             pyhd8ed1ab_1    conda-forge
sqlite                    3.49.1               h2e4c9dc_2    conda-forge
stack_data                0.6.3              pyhd8ed1ab_1    conda-forge
statsmodels               0.14.4          py312h3a11e2b_0    conda-forge
svt-av1                   3.0.1                h240833e_0    conda-forge
tblib                     3.0.0              pyhd8ed1ab_1    conda-forge
termcolor                 2.5.0              pyhd8ed1ab_1    conda-forge
threadpoolctl             3.6.0              pyhecae5ae_0    conda-forge
tifffile                  2024.12.12         pyhd8ed1ab_0    conda-forge
tinycss2                  1.4.0              pyhd8ed1ab_0    conda-forge
tk                        8.6.13               h1abcd95_1    conda-forge
toml                      0.10.2             pyhd8ed1ab_1    conda-forge
tomli                     2.2.1              pyhd8ed1ab_1    conda-forge
tomlkit                   0.13.2             pyha770c72_1    conda-forge
toolz                     1.0.0              pyhd8ed1ab_1    conda-forge
tornado                   6.4.2           py312h01d7ebd_0    conda-forge
tqdm                      4.67.1             pyhd8ed1ab_1    conda-forge
traitlets                 5.14.3             pyhd8ed1ab_1    conda-forge
trove-classifiers         2025.3.19.19       pyhd8ed1ab_0    conda-forge
typing-extensions         4.12.2               hd8ed1ab_1    conda-forge
typing_extensions         4.12.2             pyha770c72_1    conda-forge
tzdata                    2025b                h78e105d_0    conda-forge
udunits2                  2.2.28               h516ac8c_3    conda-forge
ujson                     5.10.0          py312h5861a67_1    conda-forge
ukkonen                   1.0.1           py312hc5c4d5f_5    conda-forge
unicodedata2              16.0.0          py312h01d7ebd_0    conda-forge
untokenize                0.1.1              pyhd8ed1ab_2    conda-forge
uriparser                 0.9.8                h6aefe2f_0    conda-forge
url-normalize             1.4.3              pyhd8ed1ab_1    conda-forge
urllib3                   2.3.0              pyhd8ed1ab_0    conda-forge
virtualenv                20.29.3            pyhd8ed1ab_0    conda-forge
vprof                     0.38                     pypi_0    pypi
wcwidth                   0.2.13             pyhd8ed1ab_1    conda-forge
webencodings              0.5.1              pyhd8ed1ab_3    conda-forge
webob                     1.8.9              pyhd8ed1ab_1    conda-forge
wheel                     0.45.1             pyhd8ed1ab_1    conda-forge
wrapt                     1.17.2          py312h01d7ebd_0    conda-forge
x265                      3.5                  hbb4e6a2_3    conda-forge
xarray                    2025.3.0           pyhd8ed1ab_0    conda-forge
xerces-c                  3.2.5                h197e74d_2    conda-forge
xesmf                     0.8.8              pyhd8ed1ab_1    conda-forge
xgboost                   2.1.4           cpu_pyhc1a9e11_1    conda-forge
xlsxwriter                3.2.2              pyhd8ed1ab_0    conda-forge
xorg-libxau               1.0.12               h6e16a3a_0    conda-forge
xorg-libxdmcp             1.1.5                h00291cd_0    conda-forge
xxhash                    0.8.3                h6e16a3a_0    conda-forge
xyzservices               2025.1.0           pyhd8ed1ab_0    conda-forge
yamale                    5.3.0              pyhd8ed1ab_0    conda-forge
yaml                      0.2.5                h0d85af4_2    conda-forge
yamllint                  1.35.1             pyhd8ed1ab_1    conda-forge
yarl                      1.18.3          py312h3520af0_1    conda-forge
zarr                      3.0.6              pyhd8ed1ab_0    conda-forge
zeromq                    4.3.5                h7130eaa_7    conda-forge
zfp                       1.0.1                h469392a_2    conda-forge
zict                      3.0.0              pyhd8ed1ab_1    conda-forge
zipp                      3.21.0             pyhd8ed1ab_1    conda-forge
zlib                      1.3.1                hd23fc13_2    conda-forge
zlib-ng                   2.2.4                h04d1b7c_0    conda-forge
zstandard                 0.23.0          py312h01d7ebd_1    conda-forge
zstd                      1.5.7                h8210216_2    conda-forge
```

### Environment info

```shell
architecture: x64
    activate-environment: esmvaltool
    environment-file: environment_osx.yml
    python-version: 3.12
    miniforge-version: latest
    use-mamba: true
    mamba-version: 2.0.5
    auto-activate-base: true
    auto-update-conda: false
    remove-profiles: true
    conda-solver: libmamba
    clean-patched-environment-file: true
    run-post: true
```

--- a/.ci_support/osx_64_numpy2.0python3.10.____cpython.yaml
+++ b/.ci_support/osx_64_numpy2.0python3.10.____cpython.yaml
@@ -14,6 +14,8 @@ cxx_compiler:
 - clangxx
 cxx_compiler_version:
 - '18'
+libabseil:
+- '20240722'
 macos_machine:
 - x86_64-apple-darwin13.4.0
 numpy:

--- a/.ci_support/osx_64_numpy2.0python3.11.____cpython.yaml
+++ b/.ci_support/osx_64_numpy2.0python3.11.____cpython.yaml
@@ -14,6 +14,8 @@ cxx_compiler:
 - clangxx
 cxx_compiler_version:
 - '18'
+libabseil:
+- '20240722'
 macos_machine:
 - x86_64-apple-darwin13.4.0
 numpy:

--- a/.ci_support/osx_64_numpy2.0python3.12.____cpython.yaml
+++ b/.ci_support/osx_64_numpy2.0python3.12.____cpython.yaml
@@ -14,6 +14,8 @@ cxx_compiler:
 - clangxx
 cxx_compiler_version:
 - '18'
+libabseil:
+- '20240722'
 macos_machine:
 - x86_64-apple-darwin13.4.0
 numpy:

--- a/.ci_support/osx_64_numpy2.0python3.9.____cpython.yaml
+++ b/.ci_support/osx_64_numpy2.0python3.9.____cpython.yaml
@@ -14,6 +14,8 @@ cxx_compiler:
 - clangxx
 cxx_compiler_version:
 - '18'
+libabseil:
+- '20240722'
 macos_machine:
 - x86_64-apple-darwin13.4.0
 numpy:

--- a/.ci_support/osx_64_numpy2python3.13.____cp313.yaml
+++ b/.ci_support/osx_64_numpy2python3.13.____cp313.yaml
@@ -14,6 +14,8 @@ cxx_compiler:
 - clangxx
 cxx_compiler_version:
 - '18'
+libabseil:
+- '20240722'
 macos_machine:
 - x86_64-apple-darwin13.4.0
 numpy:

--- a/.ci_support/osx_arm64_numpy2.0python3.10.____cpython.yaml
+++ b/.ci_support/osx_arm64_numpy2.0python3.10.____cpython.yaml
@@ -14,6 +14,8 @@ cxx_compiler:
 - clangxx
 cxx_compiler_version:
 - '18'
+libabseil:
+- '20240722'
 macos_machine:
 - arm64-apple-darwin20.0.0
 numpy:

--- a/.ci_support/osx_arm64_numpy2.0python3.11.____cpython.yaml
+++ b/.ci_support/osx_arm64_numpy2.0python3.11.____cpython.yaml
@@ -14,6 +14,8 @@ cxx_compiler:
 - clangxx
 cxx_compiler_version:
 - '18'
+libabseil:
+- '20240722'
 macos_machine:
 - arm64-apple-darwin20.0.0
 numpy:

--- a/.ci_support/osx_arm64_numpy2.0python3.12.____cpython.yaml
+++ b/.ci_support/osx_arm64_numpy2.0python3.12.____cpython.yaml
@@ -14,6 +14,8 @@ cxx_compiler:
 - clangxx
 cxx_compiler_version:
 - '18'
+libabseil:
+- '20240722'
 macos_machine:
 - arm64-apple-darwin20.0.0
 numpy:

--- a/.ci_support/osx_arm64_numpy2.0python3.9.____cpython.yaml
+++ b/.ci_support/osx_arm64_numpy2.0python3.9.____cpython.yaml
@@ -14,6 +14,8 @@ cxx_compiler:
 - clangxx
 cxx_compiler_version:
 - '18'
+libabseil:
+- '20240722'
 macos_machine:
 - arm64-apple-darwin20.0.0
 numpy:

--- a/.ci_support/osx_arm64_numpy2python3.13.____cp313.yaml
+++ b/.ci_support/osx_arm64_numpy2python3.13.____cp313.yaml
@@ -14,6 +14,8 @@ cxx_compiler:
 - clangxx
 cxx_compiler_version:
 - '18'
+libabseil:
+- '20240722'
 macos_machine:
 - arm64-apple-darwin20.0.0
 numpy:

--- a/.ci_support/win_64_numpy2.0python3.10.____cpython.yaml
+++ b/.ci_support/win_64_numpy2.0python3.10.____cpython.yaml
@@ -6,6 +6,8 @@ channel_targets:
 - conda-forge main
 cxx_compiler:
 - vs2019
+libabseil:
+- '20240722'
 numpy:
 - '2.0'
 pin_run_as_build:

--- a/.ci_support/win_64_numpy2.0python3.11.____cpython.yaml
+++ b/.ci_support/win_64_numpy2.0python3.11.____cpython.yaml
@@ -6,6 +6,8 @@ channel_targets:
 - conda-forge main
 cxx_compiler:
 - vs2019
+libabseil:
+- '20240722'
 numpy:
 - '2.0'
 pin_run_as_build:

--- a/.ci_support/win_64_numpy2.0python3.12.____cpython.yaml
+++ b/.ci_support/win_64_numpy2.0python3.12.____cpython.yaml
@@ -6,6 +6,8 @@ channel_targets:
 - conda-forge main
 cxx_compiler:
 - vs2019
+libabseil:
+- '20240722'
 numpy:
 - '2.0'
 pin_run_as_build:

--- a/.ci_support/win_64_numpy2.0python3.9.____cpython.yaml
+++ b/.ci_support/win_64_numpy2.0python3.9.____cpython.yaml
@@ -6,6 +6,8 @@ channel_targets:
 - conda-forge main
 cxx_compiler:
 - vs2019
+libabseil:
+- '20240722'
 numpy:
 - '2.0'
 pin_run_as_build:

--- a/.ci_support/win_64_numpy2python3.13.____cp313.yaml
+++ b/.ci_support/win_64_numpy2python3.13.____cp313.yaml
@@ -6,6 +6,8 @@ channel_targets:
 - conda-forge main
 cxx_compiler:
 - vs2019
+libabseil:
+- '20240722'
 numpy:
 - '2'
 pin_run_as_build:

--- a/.gitignore
+++ b/.gitignore
@@ -25,3 +25,5 @@
 
 # Rattler-build's artifacts are in `output` when not specifying anything.
 /output
+# Pixi's configuration
+.pixi

--- a/recipe/meta.yaml
+++ b/recipe/meta.yaml
@@ -12,7 +12,7 @@ source:
     - 0001-fix-cmake.patch
 
 build:
-  number: 2
+  number: 3
   skip: true  # [py<39]
   script: {{ PYTHON }} -m pip install . --no-deps -vv --ignore-installed --no-cache-dir --no-build-isolation
 

--- a/recipe/meta.yaml
+++ b/recipe/meta.yaml
@@ -34,6 +34,7 @@ requirements:
     - scikit-build-core
     - numpy
     - s2geometry >=0.10.0
+    - libabseil  # [not linux]
   run:
     - python
     - {{ pin_compatible('s2geometry', max_pin='x.x') }}

